shared/profile/user/index.js

```diff
@@ -39,7 +39,7 @@ export type Props = {|
   username: string,
 |}
 
-const colorTypeToStyle = type => {
+export const colorTypeToStyle = (type: 'red' | 'green' | 'blue') => {
```

Comment:
`BackgroundColorType`

Reply:
actually not exported anymore so i don't need the type